### PR TITLE
Jeppison Newmuni Compatability [Ready]

### PR DESCRIPTION
**Pull request by Omppusolttu**
## About The Pull Request

Makes the jeppison compatible with Newmunitions

## Why It's Good For The Game

Makes one of our ships better with the parts of my soul torn out by github

## Changelog
:cl:
dd: 3 .50 Cal guns and 2 Deck guns to the Jeppison
add: Maintenance around chapel in the Jeppison
add: A powder magazine and a Munitions techlathe to the Jeppison
del: Removed Maintenance around arrivals in the Jeppison
/:cl:

Fixes: https://github.com/BeeStation/NSV13/issues/1181

### CODE REVIEW

**Comment by Vasily2013** [comment]
DON'T. CHANGE. THE .DME

File: nsv13.dme [hunk 1]
```diff
@@ -14,6 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\_basemap.dm"
+#include "_maps\jeppison.dm"
```
Comment (Kmc2000) [inline nsv13.dme:17]: This PR changes the DME to force jeppison only

**Review by Kmc2000 — DISMISSED** [review]

File: nsv13.dme (hunk 1 above)
Comment (Omppusolttu) [reply nsv13.dme:17]: how the fuck did THAT happen?

Comment (Omppusolttu) [reply nsv13.dme:17]: I have no fucking clue how to fix this shit. I never even touched code- oh god, when testing I only checkboxed the jeppison as a map so I could test it...

**Comment by Omppusolttu** [comment]
Alright, that should fix it

**Comment by Jalleo** [comment]
I edited your comment to say that it fixes https://github.com/BeeStation/NSV13/issues/1181 please confirm that it does. If not remove that label until it does.

**Comment by Omppusolttu** [comment]
This does fix #1181 yes.

**Review by Vasily2013 — CHANGES_REQUESTED** [review]
image
Power cable got disconnected.

**Review by Vasily2013 — APPROVED** [review]
I don't see anything wrong.

**Pull request merged**